ads/google/a4a/test/test-utils.js

```diff
@@ -96,4 +121,55 @@ describe('Google A4A utils', () => {
         '3,4,1,2,11,12,5,6,100px,101px');
     });
   });
+
+  describe('#injectActiveViewAmpAnalyticsElement', () => {
```

Comment:
If you're writing this test from scratch, you should probably use the new `describes` helpers:

```javascript
describes.sandboxed('#injectActiveViewAmpAnalyticsElement', {}, () => {
  describes.fakeWin('some name', {amp: true}, env => {
    const win = env.win;
    // tests that can access sandbox and win
  }
}
```

I _think_ that will provide you everything that `setupForAdTesting` is currently giving you, as well as most of what `createIframePromise` gives you.